react/utils/withUsePixel.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { usePixel } from "vtex.pixel-manager/PixelContext";
+
+// tslint:disable-next-line:variable-name
+export default function withUsePixel(ComponentWrapped: typeof React.Component) {
```

Comment:
There is a `withPixel` implementation on pixel-manager that we can use:
https://github.com/vtex/pixel-manager/blob/master/react/PixelContext.tsx#L67-L79

Reply:
Done!